agenta-web/cypress/e2e/testset.cy.ts

```diff
@@ -40,4 +40,187 @@ describe("create a new testset", () => {
                 })
             })
     })
+
+    it("successfully creates the testset and adds options to it", () => {
+        const testsetName = randString(8)
+
+        const countries = [
+            {country: "USA", capital: "Washington DC", dummy: "Stuff"},
+            {country: "Nigeria", capital: "Abuja", dummy: "Thing"},
+            {country: "Egypt", capital: "Cairo", dummy: "Nothing"},
+            {country: "Ethopia", capital: "Addis Ababa", dummy: "Infinity"},
+        ]
+        cy.get(".ag-root-wrapper").as("grid")
+
+        // set values for the cells
```

Comment:
remove the comments and separate each test in an `it()` block
E.g
it("When setting values for the cells", () => {
...
}